### PR TITLE
Blog for removing cross-pod unix domain sockets

### PR DESCRIPTION
**Pull request by lei-tang**
Please provide a description for what this PR is for: Blog for removing cross-pod unix domain sockets.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[X ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[X ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/blog/2020/istio-agent/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Remove cross-pod unix domain sockets
+description: A more secure way to manage secrets.
+publishdate: 2020-02-20
+attribution: Lei Tang (Google)
+keywords: [security, secret discovery service, unix domain socket]
+target_release: 1.5
+---
+
+In Istio versions before 1.5, during secret discovery service (SDS),
```
Comment (frankbu) [inline content/en/blog/2020/istio-agent/index.md]:
```suggestion
In Istio versions before 1.5, during secret discovery service (SDS) execution,
```

File: content/en/blog/2020/istio-agent/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+the SDS client and the SDS server communicate through a cross-pod Unix domain
+socket (UDS), which needs to be protected by Kubernetes pod security policies.
+
+With Istio 1.5, Pilot Agent, Envoy, and Citadel Agent will be running in
+the same container (the architecture is in the following diagram).
```
Comment (frankbu) [inline content/en/blog/2020/istio-agent/index.md]:
```suggestion
the same container (the architecture is shown in the following diagram).
```

File: content/en/blog/2020/istio-agent/index.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+To defend attackers eavesdropping on the cross-pod UDS between Envoy (SDS client)
```
Comment (frankbu) [inline content/en/blog/2020/istio-agent/index.md]:
```suggestion
To defend against attackers eavesdropping on the cross-pod UDS between Envoy (SDS client)
```

File: content/en/blog/2020/istio-agent/index.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+and Citadel Agent (SDS server), Istio 1.5 merges Pilot Agent and Citadel Agent
+as Istio Agent and makes the UDS between Envoy and Citadel Agent
```
Comment (frankbu) [inline content/en/blog/2020/istio-agent/index.md]:
```suggestion
into a single Istio Agent and makes the UDS between Envoy and Citadel Agent
```

File: content/en/blog/2020/istio-agent/index.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+be private to the Istio Agent container.
```
Comment (frankbu) [inline content/en/blog/2020/istio-agent/index.md]:
```suggestion
private to the Istio Agent container.
```

File: content/en/blog/2020/istio-agent/index.md (hunk 4 above)
Comment (frankbu) [inline content/en/blog/2020/istio-agent/index.md]:
```suggestion
into a single Istio Agent and makes the UDS between Envoy and Citadel Agent
```

File: content/en/blog/2020/istio-agent/index.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+Istio Agent container is deployed as the sidecar for the backend container.
```
Comment (frankbu) [inline content/en/blog/2020/istio-agent/index.md]:
```suggestion
The Istio Agent container is deployed as the sidecar of the application service container.
```

File: content/en/blog/2020/istio-agent/index.md (hunk 1 above)
Comment (lei-tang) [reply content/en/blog/2020/istio-agent/index.md]: Done.

File: content/en/blog/2020/istio-agent/index.md (hunk 2 above)
Comment (lei-tang) [reply content/en/blog/2020/istio-agent/index.md]: Done.

File: content/en/blog/2020/istio-agent/index.md (hunk 3 above)
Comment (lei-tang) [reply content/en/blog/2020/istio-agent/index.md]: Done.

File: content/en/blog/2020/istio-agent/index.md (hunk 5 above)
Comment (lei-tang) [reply content/en/blog/2020/istio-agent/index.md]: Done.

File: content/en/blog/2020/istio-agent/index.md (hunk 4 above)
Comment (lei-tang) [reply content/en/blog/2020/istio-agent/index.md]: Done.

File: content/en/blog/2020/istio-agent/index.md (hunk 6 above)
Comment (lei-tang) [reply content/en/blog/2020/istio-agent/index.md]: Done.

**Pull request merged**